customer-commons/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     implementation libs.aws.sdk2.dynamodb
     implementation libs.aws.sdk2.urlconnection
 
+    testImplementation 'io.github.ganadist.sqlite4java:libsqlite4java-osx-aarch64:1.0.392'
```

Comment:
is this necessary? everything builds without it.

Reply:
removed